prac_09/sort_files_1.py

```diff
@@ -0,0 +1,63 @@
+"""CP1404 - Prac 09
+Sort files program
+Author: Zeb del Rosario
+Start date: 13/05/2021
+"""
+import os
+import shutil
+
+ROOT_DIRECTORY = os.getcwd() + "\\" + "FilesToSort"  # force cwd to be set to "FilesToSort"
+
+
+def main():
```

Comment:
Wowza! This would have taken a while. Try to use os.mkdir() to create a directory so the code is short and more efficient. There are many classes when you only really need one main class.